app/controllers/UpfrontPaymentController.scala

```diff
@@ -112,18 +121,37 @@ class UpfrontPaymentController @Inject() (
 
 object UpfrontPaymentController {
 
-  def canYouMakeAnUpfrontPaymentQuestionForm: Form[String] = Form(
-    mapping(
-      "CanYouMakeAnUpFrontPayment" -> nonEmptyText
-    )(identity)(Some(_))
-  )
+  sealed trait CanMakeUpfrontPayment extends enumeratum.EnumEntry {
```

Comment:
Some basic model which will be used in form.

Reply:
the name here can get confusing with the root `CanPayUpfront`model - maybe we could adopt some convention like `CanPayUpfrontFormValue` as a name for this sealed trait here and beyond?

Reply:
Yes, that makes sense.

Reply:
Done

Reply:
:+1: thanks